### PR TITLE
Improve values defaults and schema

### PR DESCRIPTION
**Pull request by vxav**
This pr cleans up the values files, fixes some parts of the schema and sets additional defaults

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

File: helm/cluster-vsphere/values.yaml [hunk 1]
```diff
@@ -142,152 +70,61 @@ internal:
...
-      #   # <integer> -required-
-      # - metric: 50
-      #   # To is an IPv4 or IPv6 address.
-      #   # <string> -required-
-      #   to: ""
-      #   # Via is an IPv4 or IPv6 address.
-      #   # <string> -required-
-      #   via: ""
-
+      - networkName: 'grasshopper-capv'
```
Comment (glitchcrab) [inline helm/cluster-vsphere/values.yaml:91]: is it worth removing this default from the chart? this is very specific to our infra and wouldn't work for anyone else

Comment (vxav) [reply helm/cluster-vsphere/values.yaml:91]: Good point

Comment (vxav) [reply helm/cluster-vsphere/values.yaml]: https://github.com/giantswarm/cluster-vsphere/pull/177/commits/ef1f8da4e7eeb44eb93b84fa1b611aed05752613

**Comment by vxav** [comment]
Adding `skip-ci`, the tests already succeeded.

**Review by glitchcrab — APPROVED** [review]
there's a lot going on in this PR but it seems to be ok, the values schema validates just fine

**Pull request merged**